config/config.go

```diff
@@ -58,90 +40,19 @@ type Config struct {
 	BackendsConfig  json.RawMessage `json:"backends,omitempty"`
 	TasksConfig     json.RawMessage `json:"tasks,omitempty"`
 	TelemetryConfig json.RawMessage `json:"telemetry,omitempty"`
```

Comment:
Not necessarily in scope for this PR, but `omitempty` is really only used when we serialize a struct, right? Maybe we don't really need this annotation.